### PR TITLE
added scrape script for scraping

### PR DESCRIPTION
**Pull request by VijayKrish93**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,16 +28,15 @@
...
     "webpack": "^3.0.0",
     "webpack-dev-server": "^2.5.0"
   },
   "scripts": {
     "sequelize": "sequelize",
     "sequelize:debug": "cross-env DEBUG=\"sequelize*\" sequelize",
-    "scrape": "node -e \"require('./src/server/services/scraper').scrape()\"",
-    "scrape:users": "node -e \"require('./src/server/services/scraper/users').scrape()\"",
-    "scrape:repos": "node -e \"require('./src/server/services/scraper/repos').scrape()\"",
+    "scrape":"src/server/scripts/scraper.js",
```
Comment (rmariuzzo) [inline package.json]: @VijayKrish93 please add a space after the `:`.

Comment (rmariuzzo) [reply package.json]: And also prepend the file path with `./` to follow the same existing style.

File: src/server/scripts/scraper.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+//destructing the command line arguments
+const [,,...args]=process.argv;
```
Comment (rmariuzzo) [inline src/server/scripts/scraper.js]: Please add a space between and after the `=`. And we are not using `;`, please remove it here and in the other lines.

File: src/server/scripts/scraper.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+switch(args[0]){
```
Comment (rmariuzzo) [inline src/server/scripts/scraper.js]: Add a space before the `{`

File: src/server/scripts/scraper.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	case '--only=users':
+	  console.log('fetching only users repos ===============');
```
Comment (rmariuzzo) [inline src/server/scripts/scraper.js]: Please, remove the `console.log` statements.

**Comment by VijayKrish93** [comment]
@rmariuzzo  The requested changes are done and the PR is updated.I would like to contribute more to this project.Is there a road map guide for the project where you have listed down the future requirements?

**Pull request merged**